src/common/Tag.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from "../theme"
+
+export const StyledTag = styled.span(() => ({
+  margin: 2,
+  padding: 4,
+  borderRadius: 5,
+  lineHeight: 1,
+  border: "1px solid green",
```

Comment:
I am not sure about this actually... We have a proper design system setup in the project, would be cool to follow it 🙂  Especially because there is dark mode, it would be cool to make it look nice there too. But we can do it later in a different PR!

Reply:
Sounds good.